generators/server/templates/dotnetcore/src/Project/Configuration/DatabaseStartup.cs.ejs

```diff
@@ -61,7 +61,7 @@ public static class DatabaseConfiguration
                     connectionString = $"Server={databaseUri.Host};Port={databaseUri.Port};Database={db};User={userInfo[0]};Password={userInfo[1]};Pooling=true;MinimumPoolSize=0;MaximumPoolsize=10;";
                     break;
                 case "mssql":
-                    connectionString = $"Server={databaseUri.Host};Port={databaseUri.Port};Database={db};User={userInfo[0]};Password={userInfo[1]};Trusted_Connection=False;Pooling=true;";
+                    connectionString = $"Server={databaseUri.Host};Port={databaseUri.Port};Database={db};User={userInfo[0]};Password={userInfo[1]};Trusted_Connection=False;Encrypt=False;Pooling=true;";
```

Comment:
Why disable encryption ? It's by default now ?

Reply:
Yep, I worked with some project .Net 7 now and without this setting, we have to check "Trust server certificate" in SQL Server or it'll show the error of certificate when open the connection to SQL Server
![image](https://user-images.githubusercontent.com/4284170/229376256-818cadcb-fd72-4dba-878a-029976485e2b.png)

Reply:
Or we can use the setting ";Encrypt=True;TrustServerCertificate=True" for the same way.

Reply:
@nicolas63 I've updated .NET 7 for jhipster-net . I can make a PR for it if you want

Reply:
Yes please

Reply:
Ah I did , there is https://github.com/jhipster/jhipster-net/pull/14 ^^!